rc/windowing/new-client.kak

```diff
@@ -3,7 +3,15 @@ new [<commands>]: create a new Kakoune client
 The ''terminal'' alias is being used to determine the user''s preferred terminal emulator
 The optional arguments are passed as commands to the new client' \
 %{
-    terminal kak -c %val{session} -e "%arg{@}"
+
+    evaluate-commands %sh{
+        if [ $# -gt 0 ]; then
+            echo "echo -debug '$@'"
+            echo "terminal kak -c %val{session} -e \"$@\""
+        else
+            echo "terminal kak -c %val{session}"
```

Comment:
> I did that because an empty `:new` would pass nothing to `kak -e`, making it exit.

`:new` works for me, this must be a problem in your `terminal` implementation.
Instead of

	tilix -a app-new-window -a kak -c session

we probably need something to the effect of

	tilix -a app-new-window -a "kak -c session"